### PR TITLE
Remove video reader warning that a frame has been seen twice

### PR DESCRIPTION
**Pull request by JanuszL**
- the NVVL that the DALI video reader is based on, reports
  a warning when a video packet with the same timestamp has
  been seen already
- recently introduced heuristics allows the reader to rewind
  to the previous key frame as FFmpeg doesn't differentiate
  between IDR and non-IDR keyframes, while NVDEC cannot resume
  decoding from a non-IDR keyframe. This functionality makes
  the mentioned warning an expected behavior in some usaceses

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- the NVVL that the DALI video reader is based on, reports
  a warning when a video packet with the same timestamp has
  been seen already
- recently introduced heuristics allows the reader to rewind
  to the previous key frame as FFmpeg doesn't differentiate
  between IDR and non-IDR keyframes, while NVDEC cannot resume
  decoding from a non-IDR keyframe. This functionality makes
  the mentioned warning an expected behavior in some usaceses

### Affected modules and functionalities:
- video reader

### Key points relevant for the review:
- NA

### Tests:

- [x] Existing tests apply
  - test_video_pipeline.py
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [x] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Review by awolant — APPROVED** [review]

File: dali/operators/reader/loader/video_loader.cc [hunk 1]
```diff
@@ -607,18 +605,6 @@ void VideoLoader::read_file() {
         continue;
     }
 
-    if (frame >= req.frame && frame < req.frame + req.count) {
-      if (frames_read[frame - req.frame]) {
-        ERROR_LOG << "Frame " << frame << " appeared twice\n";
-      } else {
-        frames_read[frame - req.frame] = true;
-        frames_left--;
```
Comment (mzient) [inline dali/operators/reader/loader/video_loader.cc:615]: What about this piece?

Comment (JanuszL) [reply dali/operators/reader/loader/video_loader.cc:615]: Same. As the reader can rewind this part seems to be no longer relevant.

Comment (mzient) [reply dali/operators/reader/loader/video_loader.cc:615]: I trust you that it's tested and works.

**Review by mzient — APPROVED** [review]

File: dali/operators/reader/loader/video_loader.cc (hunk 1 above)
Comment (JanuszL) [reply dali/operators/reader/loader/video_loader.cc:615]: This logic is only for logging. I don't see it used for the actual flow.

**Comment by JanuszL** [comment]
!build

**Pull request merged**